### PR TITLE
Update proc_creation_macos_xattr_gatekeeper_bypass.yml

### PR DESCRIPTION
**Pull request by pratinavchandra**
### Summary of the Pull Request
Updating rule proc_creation_macos_xattr_gatekeeper_bypass.yml

The current rule checks for the wrong option. The actual option that deletes the attribute is `-d`
The current rule will miss cases where the quarantine attribute is removed using only the `-d` option.
`xattr -d com.apple.quarantine FILE`

Checking for only the `-d` option also covers when attributes are removed recursively using `xattr -d -r com.apple.quarantine /path/`

man page for xattr:
image

### Changelog

update: Gatekeeper Bypass via Xattr - Update command line flag

### Example Log Event
image

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/macos/process_creation/proc_creation_macos_xattr_gatekeeper_bypass.yml [hunk 1]
```diff
@@ -4,9 +4,10 @@ status: test
 description: Detects macOS Gatekeeper bypass via xattr utility
 references:
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1553.001/T1553.001.md
-author: Daniil Yugoslavskiy, oscd.community
+    - https://www.loobins.io/binaries/xattr/
+author: Daniil Yugoslavskiy, oscd.community, Pratinav Chandra
```
Comment (nasbench) [inline rules/macos/process_creation/proc_creation_macos_xattr_gatekeeper_bypass.yml:8]:
```suggestion
author: Daniil Yugoslavskiy, oscd.community
```

File: rules/macos/process_creation/proc_creation_macos_xattr_gatekeeper_bypass.yml [hunk 2]
```diff
@@ -4,9 +4,10 @@ status: test
 description: Detects macOS Gatekeeper bypass via xattr utility
 references:
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1553.001/T1553.001.md
```
Comment (nasbench) [inline rules/macos/process_creation/proc_creation_macos_xattr_gatekeeper_bypass.yml:6]:
```suggestion
    - https://github.com/redcanaryco/atomic-red-team/blob/1fed40dc7e48f16ed44dcdd9c73b9222a70cca85/atomics/T1553.001/T1553.001.md
```

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
Thanks for the contribution @pratinavchandra 
Unfortunately you can't claim authorship of a rule for just fixing a small typo. You will be credited in the release section in the coming days

Thanks :)

**Comment by pratinavchandra** [comment]
Oh okay! In my view, I didn't think of it as just a typo, I thought of it as a logical error in which command line option to detect. I understand though :)
Thanks!

**Pull request merged**